src/main/java/dudu/command/Command.java

```diff
@@ -0,0 +1,19 @@
+package dudu.command;
```

Comment:
I like how you split all the commands into individual classes for better readability.